src/app/components/feed/feed-connection.factory.js

```diff
@@ -120,19 +115,10 @@
             console.log(jsep);
             var start = { "request": "start", "room": roomId };
             pluginHandle.send({message: start, jsep: jsep});
-            // Is it ok to send the configure without waiting for
-            // the response to the start command?
-            var config = {audio: true, video: false};
-            if (options.withVideo) { config.video = true; }
-            that.config = new ConnectionConfig(pluginHandle, config, jsep);
-            // Call the provided callback for extra actions
-            if (options.success) { options.success(); }
```

Comment:
I'd prefer something like `typeof options.success === "function"`. Anyway, it's fine with me.